scripts/build_spec.py

```diff
@@ -25,7 +25,7 @@
     signing_root,
 )
 from eth2spec.utils.ssz.ssz_typing import (
-    Bit, Bool, Container, List, Vector, Bytes, uint64,
+    bit, boolean, Container, List, Vector, Bytes, uint64,
```

Comment:
Why not be consistent with casing, i.e. `bit` => `Bit` and `boolean` => `Boolean`? Also why `Boolean` instead of the "standard" `Bool`?

Reply:
My guess would be lower-case for basic types, and upper-case for non-basic types. In which case this makes sense :)

Reply:
Yes, I think this is consistent with typing, just like uintN are lowercase. So I'm using "boolean" because bool is already a type in python.